web/src/main/java/io/fabric8/launcher/web/endpoints/BoosterCatalogEndpoint.java

```diff
@@ -42,6 +43,9 @@
     @Inject
     private BoosterCatalogFactory boosterCatalogFactory;
 
+    @Inject
+    private BoosterDocumentationStore boosterDocumentationStore;
```

Comment:
I think this should belong in a separate endpoint

Reply:
Look at the issue, I followed @quintesse idea there

Reply:
I think what he meant was to do something similar, It doesn't make sense to trigger the readme index in this endpoint

Reply:
No problem here with making it a separate endpoint.